docs/reference/components/sources/aws_s3.cue

```diff
@@ -0,0 +1,225 @@
+package metadata
+
+components: sources: aws_s3: {
+	title:       "AWS S3"
+	description: "[Amazon Simple Storage Service (Amazon S3)][urls.aws_s3] is a scalable, high-speed, web-based cloud storage service designed for online backup and archiving of data and applications on Amazon Web Services. It is very commonly used to store log data."
+
+	features: {
+		multiline: enabled: true
+		collect: {
+			tls: enabled:        false
+			checkpoint: enabled: false
+			from: {
+				name:     "AWS S3"
+				thing:    "an \(name) bucket"
+				url:      urls.aws_s3
+				versions: null
+
+				setup: [
+					"""
+						Create an [AWS SQS queue][urls.aws_sqs] for Vector to consume bucket
+						notifications from. Then, configure the [bucket
+						notifications](https://docs.aws.amazon.com/AmazonS3/latest/dev/ways-to-add-notification-config-to-bucket.html)
+						to publish to this queue for the following events:
+
+						- PUT
+						- POST
+						- COPY
+						- Multipart upload completed
+
+						These represent object creation events.
+						""",
+				]
+			}
+		}
+	}
+
+	classes: {
+		commonly_used: true
+		deployment_roles: ["aggregator"]
+		delivery:      "at_least_once"
+		development:   "beta"
+		egress_method: "stream"
+	}
+
+	support: {
+		platforms: {
+			"aarch64-unknown-linux-gnu":  true
+			"aarch64-unknown-linux-musl": true
+			"x86_64-apple-darwin":        true
+			"x86_64-pc-windows-msv":      true
+			"x86_64-unknown-linux-gnu":   true
+			"x86_64-unknown-linux-musl":  true
+		}
+
+		requirements: []
+		warnings: []
+		notices: []
+	}
+
+	configuration: {
+		strategy: {
+			common:      true
```

Comment:
Should we set this to `false`, given that there isn't any strategy other than the default to pick at the moment?